igor-web/src/main/java/com/netflix/spinnaker/igor/gcb/GoogleCloudBuildController.java

```diff
@@ -86,4 +88,25 @@ Build getBuild(@PathVariable String account, @PathVariable String buildId) {
     Build build = googleCloudBuildParser.parse(serializedBuild, Build.class);
     return googleCloudBuildAccountRepository.getGoogleCloudBuild(account).extractArtifacts(build);
   }
+
+  @RequestMapping(value = "/triggers/{account}", method = RequestMethod.GET)
+  @PreAuthorize("hasPermission(#account, 'BUILD_SERVICE', 'READ')")
+  List<BuildTrigger> listTriggers(@PathVariable String account) {
+    return googleCloudBuildAccountRepository.getGoogleCloudBuild(account).listTriggers();
+  }
+
+  @RequestMapping(
+      value = "/triggers/run/{account}/{triggerId}",
```

Comment:
I think this mapping might be better as `/triggers/{account}/{triggerId}/run` to keep the resource identifier (`/triggers/{account}/{triggerid}`) together and move the custom verb to the end.  (This would also keep it consistent with how we mapped the `getArtifacts` custom action above, by adding the verb at the end of the resource.)

Reply:
Makes sense, will implement it that way.